inc/field/glpiselectfield.class.php

```diff
@@ -149,7 +149,7 @@ public function buildParams($rand = null) {
       if (isset($emptyItem->fields['otherserial'])) {
          $dparams['displaywith'][] = 'otherserial';
       }
-      if ($itemtype === Ticket::class && !array_search('id', $dparams['displaywith'])) {
+      if ($itemtype === \Ticket::class && !array_search('id', $dparams['displaywith'])) {
```

Comment:
It is better to use a ```use Ticket;``` statement in the top of the file

Reply:
done